### PR TITLE
[feature]New alarm button of management console

### PR DESCRIPTION
**Pull request by huangKai-2323**
### What is the purpose of the change

Add alarm button to the management console for dss 

### Related issues/PRs

Related issues: #3870
Related pr:#3869

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

File: linkis-public-enhancements/linkis-jobhistory/src/main/java/org/apache/linkis/jobhistory/restful/api/JobhistorySettingApi.java [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+      Map<String, String> extra = monitor.getExtra();
+      if (StringUtils.isBlank(extra.getOrDefault("title", "")))
+        return Message.error("title can't be empty ");
+      if (StringUtils.isBlank(extra.getOrDefault("detail", "")))
+        return Message.error("detail can't be empty ");
+    }
+    if (StringUtils.isBlank(monitor.getMonitorLevel())) {
+      return Message.error("MonitorLevel can't be empty ");
+    }
+    // 根据id获取jobInfo
```
Comment (Alexkun) [inline linkis-public-enhancements/linkis-jobhistory/src/main/java/org/apache/linkis/jobhistory/restful/api/JobhistorySettingApi.java:85]: Need English comments.

**Comment by casionone** [comment]
need add dml and ddl

**Comment by casionone** [comment]
```
2022-11-24T06:41:22.3904784Z [ERROR]   JobHistoryMapperTest.insertJobHistoryTest:80 » BadSqlGrammar 
2022-11-24T06:41:22.3906179Z ### Error updating database.  Cause: org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "observe_info" not found; SQL statement:
```

File: linkis-public-enhancements/linkis-jobhistory/src/main/java/org/apache/linkis/jobhistory/restful/api/JobhistorySettingApi.java (hunk 1 above)
Comment (huangKai-2323) [reply linkis-public-enhancements/linkis-jobhistory/src/main/java/org/apache/linkis/jobhistory/restful/api/JobhistorySettingApi.java]: done

File: linkis-dist/package/db/linkis_ddl.sql [hunk 2]
```diff
@@ -988,4 +988,8 @@ CREATE TABLE `linkis_cg_user_ip_config` (
   `bussiness_user` varchar(50) COLLATE utf8_bin NOT NULL,
   PRIMARY KEY (`id`),
   UNIQUE KEY `user_creator` (`user`,`creator`)
-) ENGINE=InnoDB AUTO_INCREMENT=0 DEFAULT CHARSET=utf8 COLLATE=utf8_bin
\ No newline at end of file
+) ENGINE=InnoDB AUTO_INCREMENT=0 DEFAULT CHARSET=utf8 COLLATE=utf8_bin
+
+
+
+ALTER TABLE `linkis_ps_job_history_group_history` ADD COLUMN `observe_info` VARCHAR(500) NULL DEFAULT NULL
```
Comment (casionone) [inline linkis-dist/package/db/linkis_ddl.sql:995]: modify  dml sql need to put at 1.3.2/linkis_dml.sql

Comment (casionone) [reply linkis-dist/package/db/linkis_ddl.sql:995]: missing end `;`

File: linkis-computation-governance/linkis-client/linkis-computation-client/src/test/java/org/apache/linkis/ujes/client/JobObserveActionTest.scala [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+import org.apache.linkis.ujes.client.request.{EmsListAction, JobExecuteAction, JobObserveAction, ResultSetAction}
+
+import java.util
+import java.util.concurrent.TimeUnit
+
+@Deprecated
+object JobObserveActionTest extends App {
+
+  val clientConfig = DWSClientConfigBuilder.newBuilder()
+    .addServerUrl("") // 改成测试gateway地址
```
Comment (casionone) [inline linkis-computation-governance/linkis-client/linkis-computation-client/src/test/java/org/apache/linkis/ujes/client/JobObserveActionTest.scala:32]: .addServerUrl("127.0.0.1:9001")  // set gateway service url

File: linkis-computation-governance/linkis-client/linkis-computation-client/src/test/java/org/apache/linkis/ujes/client/JobObserveActionTest.scala [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    .setAuthTokenValue("BML-AUTH")
+    .setDWSVersion("v1")
+    .build()
+  val client = new UJESClientImpl(clientConfig)
+  val map = new util.HashMap[String, String]()
+  map.put("title", "ss");
+  map.put("detail", "ss");
+  val build = JobObserveAction.builder()
+    .setUser("hadoop")
+    .setTaskId("73695")  // 这个需要真实的id
```
Comment (casionone) [inline linkis-computation-governance/linkis-client/linkis-computation-client/src/test/java/org/apache/linkis/ujes/client/JobObserveActionTest.scala:51]: need to use EN

**Pull request merged**